cycode/cli/files_collector/sca/npm/restore_npm_dependencies.py

```diff
@@ -0,0 +1,33 @@
+import os
+from typing import List
+
+import click
+
+from cycode.cli.files_collector.sca.base_restore_dependencies import BaseRestoreDependencies
+from cycode.cli.models import Document
+
+NPM_PROJECT_FILE_EXTENSIONS = ['.json']
+NPM_LOCK_FILE_NAME = 'package-lock.json'
+NPM_MANIFEST_FILE_NAME = 'package.json'
+OUTPUT_FILE_MANUALLY = False
+
+
+class RestoreNpmDependencies(BaseRestoreDependencies):
+    def __init__(self, context: click.Context, is_git_diff: bool, command_timeout: int) -> None:
+        super().__init__(context, is_git_diff, command_timeout, OUTPUT_FILE_MANUALLY)
+
+    def is_project(self, document: Document) -> bool:
+        return any(document.path.endswith(ext) for ext in NPM_PROJECT_FILE_EXTENSIONS)
+
+    def get_command(self, manifest_file_path: str) -> List[str]:
+        return ['npm', 'install', '--prefix', self.prepare_manifest_file_path_for_command(manifest_file_path),
+                '--package-lock-only', '--ignore-scripts', '--no-audit']
+
+    def get_lock_file_name(self) -> str:
+        return NPM_LOCK_FILE_NAME
+
+    def verify_restore_file_already_exist(self, restore_file_path: str) -> bool:
+        return os.path.isfile(restore_file_path)
+
+    def prepare_manifest_file_path_for_command(self, manifest_file_path: str) -> str:
+        return '/' + manifest_file_path.strip('/' + NPM_MANIFEST_FILE_NAME)
```

Comment:
as I mentioned, I used replace

Reply:
I am just worried about slashes. On Windows it is \, on unix it is /. In your code, it is hardcoded as / only. Pls verify. Or better write unit tests to cover it. Our unit tests are running on Windows, macOS, and Linux. 

In case this fails on Windows I suggest using this to get the system`s path separator: https://docs.python.org/3/library/os.html#os.sep